package.json

```diff
@@ -22,5 +22,12 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "sass": "^1.36.0"
+  },
+  "devDependencies": {
+    "@aerian/eslint-config": "^11.0.1",
```

Comment:
Could you please instead add: 
* @aerian/eslint-config-react - because it covers react better than the basic eslint
* @aerian/eslint-config-typescript - becase of the typescript in the project